services/user.js

```diff
@@ -380,8 +382,46 @@ class User {
     isAdmin(role) {
         return role && role === USER.ROLES.ADMIN;
     }
+    /**
+     * getOrgOwnerByOrgName
+     * @param {*} orgName 
+     * @returns user[]
+     */
+    async getOrgOwnerByOrgName(orgName) {
+
+        const orgOwner= {
+            "userStatus": USER.STATUSES.ACTIVE,
+            "role": USER.ROLES.ORG_OWNER,
+            "organization.orgName": orgName
+        };
+        return await this.userCollection.aggregate([{"$match": orgOwner}]) || [];
+    }
+
+    isValidOrThrow(conditions){
+        conditions.forEach((condition) => {
+            if (!condition) throw new Error(ERROR.NOT_LOGGED_IN);
+        });
+    }
+
+    epochLogTime() {
+        const logTime = addSeconds(getTimeNow(), config.token_timeout).toString();
+        return Date.parse(logTime);
+    }
+
+    async grantToken (_, context){
```

Comment:
Is this function similar or same as the the same function in Wesley's PR here? https://github.com/CBIIT/crdc-datahub-database-drivers/pull/44/files